resources/views/env.blade.php

```diff
@@ -58,6 +58,14 @@
 # Polr will redirect logged off users to this URL
 SETTING_INDEX_REDIRECT={{$ST_INDEX_REDIRECT}}
```

Comment:
Have you considered using this setting for the 404 redirect rather than adding a new option?